src/utils/filterScopes.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import Fuse from 'fuse.js'
+
+export const options = {
+  includeScore: true
+}
+
+const filterScopes = (
```

Comment:
I would inline the contents of this function where it's being used, given that we only have 1 usage so probably doesn't make too much sense to separate it as a module